packages/kununu-theme-v2/scss/base/mixins.scss

```diff
@@ -0,0 +1,35 @@
+@mixin text-shadow {
+  text-shadow: 0 1px 1px rgba($black, .2);
+}
+
+@mixin text-shadow-darker {
+  text-shadow: 0 1px 1px rgba($black, .5);
+}
+
+@mixin text-shadow-darkest {
+  text-shadow: 0 0 3px rgba(0, 0, 0, .5), 0 0 5px rgba(0, 0, 0, .7);
+}
+
+@mixin btn-text-shadow {
+  text-shadow: 0 1px 1px rgba(0, 0, 0, .15);
+}
+
+@mixin font-awesome-icon($icon) {
+  @include fa-icon;
+  content: $icon;
+}
+
+@mixin ku-icon {
+  display: inline-block;
+  font-family: "kununu";
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  line-height: 1;
+  text-decoration: inherit;
+  text-rendering: optimizeLegibility;
+  text-transform: none;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+  font-smoothing: antialiased;
+}
```

Comment:
maybe we could remove these font icons for now, since they are only used in the monolith

Reply:
Yeah good point! I'll remove this 🌮